forge-gui/res/cardsfolder/upcoming/zellix_sanity_flayer.txt

```diff
@@ -0,0 +1,10 @@
+Name:Zellix, Sanity Flayer
+ManaCost:2 U
+Types:Legendary Creature Horror
+PT:2/3
+K:Choose a Background
+T:Mode$ MilledAll | ValidPlayer$ Player | ValidCard$ Creature | TriggerZones$ Battlefield | Execute$ TrigToken | TriggerDescription$ Hive Mind — Whenever a player mills one or more creature cards, you create a 1/1 black Horror creature token.
+SVar:TrigToken:DB$ Token | TokenScript$ b_1_1_horror
+A:AB$ Mill | Cost$ 1 T | NumCards$ 3 | ValidTgts$ Player | TgtPrompt$ Select a player | SpellDescription$ Target player mills three cards.

```

Comment:
Probably just remove TgtPrompt - Forge can handle this one

I like the prompt to specify "target" when you are in fact targeting because that so often matters in MtG